OpenRA.Game/Graphics/SpriteFont.cs

```diff
@@ -134,7 +134,21 @@ public int2 Measure(string text)
 
 		GlyphInfo CreateGlyph(Pair<char, Color> c)
 		{
-			face.LoadChar(c.First, LoadFlags.Default, LoadTarget.Normal);
+			try
+			{
+				face.LoadChar(c.First, LoadFlags.Default, LoadTarget.Normal);
+			}
+			catch (FreeTypeException)
+			{
+				var bogusSprite = builder.Allocate(new Size(0, 0));
+				return new GlyphInfo
+				{
+					Sprite = bogusSprite,
```

Comment:
It would be cleaner to leave this as `null` and then check for null and then add a null check in `DrawText`.
You should also be able to use `Advance = 0` and `Offset = int2.Zero`